CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
--
+- Added tracing spans for application of policies and projection of events
```

Comment:
Remember that we follow the [Keep a Changelog convention](https://keepachangelog.com/en/1.0.0/), so this should be in an `Added` section
```suggestion
### Added

- Added tracing spans for application of policies and projection of events
```